### PR TITLE
feat: Add Avatar Component

### PR DESCRIPTION
**Pull request by manojasarsa**
Added Avatar Component

### CODE REVIEW

**Review by SJTGSHIVAM — APPROVED** [review]
UI looks clean and intractive.
LGTM

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+.avatar_sm {
+    height: 4em;
+    width: 4em;
+}
+
+.avatar_xs {
+    height: 2.5em;
+    width: 2.5em;
+}
```
Comment (gopalbbharadva) [inline components/avatar/avatar.css:62]: Suggestion: You should give consistent sizes for different variants of avatar.E.g. small(2rem) ,medium(5rem) and large(8rem).So the size different should same between each variant.

File: components/avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        <div class="avatar_container">
+            <h2 class="avatar_head">Avatar</h2>
+            <p class="avatar_desc">
+                The Avatar component is used to represent a user profile pictures on profile information page, on
+                navigation
+                bar, in blogs grid items. They usually found in circular shapes.
+            </p>
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=vscode&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Cascadia+Code&fs=13.5px&lh=146%25&si=false&es=2x&wm=false&code=%2540import%2520url%28%2522https%253A%252F%252Fzero-ui.netlify.app%252Fcomponents%252Favatar%252Favatar.css%2522%29%253B%250A%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fzero-ui.netlify.app%252Fcomponents%252Favatar%252Favatar.css%2522%250A%253E%250A"
+                style="width: 80vw; height: 30vh; border:0; transform: scale(1); overflow:hidden;"
```
Comment (gopalbbharadva) [inline components/avatar/avatar.html:73]: Good practice: Remove inline styles, it's a bad practice. It will give headache while debugging.

**Review by gopalbbharadva — APPROVED** [review]

File: components/avatar/avatar.css (hunk 1 above)
Comment (manojasarsa) [reply components/avatar/avatar.css:62]: Sure, I'll do that. Thanks, Gopal!

**Pull request merged**